CHANGELOG.md

```diff
@@ -2,208 +2,234 @@
 
 ## Upcoming version
 
-## [v0.15.0]
+## \[v0.16.0\]
 
 ### Added
-- [[#270](https://github.com/rust-vmm/vm-memory/pull/270)] atomic_bitmap: add capability to reset bits range
-- [[#285](https://github.com/rust-vmm/vm-memory/pull/285)] Annotated modules in lib.rs to indicate their feature
+
+- \[[#287](https://github.com/rust-vmm/vm-memory/pull/287)\] Add build support on riscv64.
+
+### Changed
+
+- \[[#297](https://github.com/rust-vmm/vm-memory/pull/297)\] ci: Enable RISC-V CI
+  - \[ca83227\] test: Reduce size of test_bitmap
+  - \[65f4dbf\] build: warn lints instead of deny
```

Comment:
I'm not entire sure we really need these in the changelog, they all seem very internal to me

Reply:
Removed 😉